### DISCUSSION TITLE
Osu lazer not lunching anymore

### DISCUSSION THREAD

**Discussion by lKyliel** [ppy/osu#16712] [Q&A] [answered]
So for about 3 days now I couldnt open Osu lazer at all (it worked before this)
Everytime i try and lunch it, nothing happens, no error codes, not even a window, nothing
Ive tried reinstalling which didnt help
Osu Stable still works

**Comment by bdach** [comment, marked as answer]
[Please provide logs to help us troubleshoot your problems](https://github.com/ppy/osu/discussions/15188)

and in the future make it a discussion, instead of opening issues, thanks. i've converted this one for you.

**Reply by lKyliel** [reply]
Ohhh im sorry about that, I didnt know.

Where exactly can i find the logs for Osu lazer? I dont see anything in the file location

**Reply by bdach** [reply]
you clicked the link, right?

![2022-01-30-181158_1354x573_scrot](https://user-images.githubusercontent.com/20418176/151709708-dd573413-9e95-4b23-89d0-125c03586a59.png)

**Reply by lKyliel** [reply]
;-;
Im guessing i should add everything
I might add that I have the same problem on my other PC


[performance.log](https://github.com/ppy/osu/files/7966457/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7966459/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7966460/updater.log)
[database.log](https://github.com/ppy/osu/files/7966461/database.log)
[network.log](https://github.com/ppy/osu/files/7966462/network.log)

**Reply by bdach** [reply]
log shows:

```
2022-01-30 16:57:21 [verbose]: Running osu 2021.1127.0.0 on .NET 5.0.11
```

and

```
2022-01-30 16:57:32 [error]: An unhandled error has occurred.
2022-01-30 16:57:32 [error]: Realms.Exceptions.RealmException: Provided schema version 12 is less than last set version 13.
2022-01-30 16:57:32 [error]: at Realms.NativeException.ThrowIfNecessary(Func`2 overrider)
2022-01-30 16:57:32 [error]: at Realms.SharedRealmHandle.Open(Configuration configuration, RealmSchema schema, Byte[] encryptionKey)
```

please upgrade game to the newest version and try again

**Reply by lKyliel** [reply]
Ok thank you i will try that :)

**Reply by lKyliel** [reply]
Thank you so much its working again
Much appreciated

**Comment by mk56-spn** [comment]
> So for about 3 days now I couldnt open Osu lazer at all (it worked before this) Everytime i try and lunch it, nothing happens, no error codes, not even a window, nothing Ive tried reinstalling which didnt help Osu Stable still works

do you have the latest version, pre realm version stopped working for me after the realm switch, probs that, and yeah it doesnt tell you anything

**Reply by lKyliel** [reply]
Ohhhh maybe its that yeah, is the new one on the same Github link to download?
